service/src/main/resources/application.yml

```diff
@@ -18,10 +18,10 @@ kafka:
     topic: alerts
 
 alert.store:
-  pluginDirectory: "storage-backends/elasticsearch/target"
+  pluginDirectory: "./storage-backends/"
   plugins:
   - name: elasticsearch
-    jarName: "elasticsearch-store-1.0.0-SNAPSHOT.jar"
+    jarName: "elasticsearch-store.jar"
```

Comment:
would this work? build generates elasticsearch-store-1.0.0-SNAPSHOT.jar right? are we renaming it to elasticsearch-store.jar